docs/how-to-guides/manage-label-based-access-control-permissions.md

```diff
@@ -0,0 +1,226 @@
+
+The fictional company's day-to-day business is ingesting new data for training machine learning models.
+Alice is the database administrator in the company, and she would like to set up label-based access control
+inside Memgraph to make data manipulation more secure.
+
+When she tries out Memgraph the first time, she is connected to a session with all privileges and is 
+able to create users and roles and grant them privileges. As a first task, she creates the admin role,
+which is automatically granted permission to write any clause and access any node or relationship.
+
+```cypher
+CREATE USER admin IDENTIFIED BY 'PaSsWoRd';
+```
+
+The user `admin` is also able to verify that she has all the privileges by writing:
+
+```cypher
+SHOW PRIVILEGES FOR admin;
+```
+
+privilege         | effective       | description                                   
+------------------|-----------------|------------------------------------------------
+CREATE            | GRANT           | GRANTED TO USER                             
+DELETE            | GRANT           | GRANTED TO USER                             
+MATCH             | GRANT           | GRANTED TO USER                             
+MERGE             | GRANT           | GRANTED TO USER                             
+SET               | GRANT           | GRANTED TO USER                             
+REMOVE            | GRANT           | GRANTED TO USER                             
+INDEX             | GRANT           | GRANTED TO USER                             
+STATS             | GRANT           | GRANTED TO USER                             
+AUTH              | GRANT           | GRANTED TO USER                             
+CONSTRAINT        | GRANT           | GRANTED TO USER                             
+DUMP              | GRANT           | GRANTED TO USER                             
+REPLICATION       | GRANT           | GRANTED TO USER                             
+READ_FILE         | GRANT           | GRANTED TO USER                             
+DURABILITY        | GRANT           | GRANTED TO USER                             
+FREE_MEMORY       | GRANT           | GRANTED TO USER                             
+TRIGGER           | GRANT           | GRANTED TO USER                             
+CONFIG            | GRANT           | GRANTED TO USER                             
+STREAM            | GRANT           | GRANTED TO USER                             
+MODULE_READ       | GRANT           | GRANTED TO USER                             
+MODULE_WRITE      | GRANT           | GRANTED TO USER                             
+WEBSOCKET         | GRANT           | GRANTED TO USER                             
+ALL LABELS        | CREATE_DELETE   | GLOBAL LABEL PERMISSION GRANTED TO USER     
+ALL EDGE_TYPES    | CREATE_DELETE   | GLOBAL EDGE_TYPE PERMISSION GRANTED TO USER 
+
+Alice can now log in as an administrator in Memgraph with her own account. From that point on,
```

Comment:
Maybe add somewhere here link to that how-to about creating users and granting permissions?
http://localhost:3000/docs/memgraph/next/how-to-guides/manage-user-privileges